how-it-works.css

```diff
@@ -0,0 +1,233 @@
+@import "base.css";
+
+:root {
+  --font-primary: "Manrope", sans-serif;
+}
+
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: var(--font-primary);
+  background-color: var(--white);
+}
+
+.how-it-works_title-bar {
+  width: 100%;
+  height: 120px;
+  margin: auto;
+  display: flex;
+  background-color: var(--primary);
+  color: var(--white);
+  font-weight: 400;
+  font-size: 0.9rem;
+  justify-content: center;
+  align-items: center;
+}
+
+.how-it-works_title-bar2 {
+  width: 100%;
+  height: 200px;
+  margin: auto;
+  display: flex;
+  color: var(--black);
+  align-items: center;
+  justify-content: center;
+}
+
+.how-it-works_title-bar2 h2 {
+  font-size: 2.2rem;
+  font-weight: 600;
+  text-align: center;
+  font-weight: bolder;
+}
+
+.how-it-works_space_bar {
+  width: 100%;
+  height: 30px;
+  margin: auto;
+  display: flex;
+  background-color: var(--primary);
+}
+
+/* Body Divs */
+
+/* Top and Bottom Div Settings */
+.how-it-works_body-top-bottom {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  color: var(--black);
+  font-size: 1.2rem;
+  flex-direction: row;
+}
+
+.how-it-works_body-top-bottom_left {
+  width: 55%;
+  margin: 40px 20px 0px 40px;
+}
+
+.how-it-works_body-top-bottom_left h3 {
+  font-weight: 600;
+  font-size: 2rem;
+}
+
+.how-it-works_body-top-bottom_left p {
+  font-weight: 400;
+  font-size: 1.8rem;
+}
+
+.how-it-works_body-top-bottom_right {
+  width: 45%;
+}
+
+.how-it-works_body-top-bottom_right img {
+  height: 600px;
+  width: 700px;
+}
+
+/* Middle Divs Setting */
+.how-it-works_body-middle {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  color: var(--black);
+  font-size: 1.2rem;
+  flex-direction: row;
+}
+
+.how-it-works_body-middle_left {
+  width: 45%;
+}
+
+.how-it-works_body-middle_left img {
+  height: 600px;
+  width: 700px;
+}
+
+.how-it-works_body-middle_right {
+  width: 55%;
+  margin: 40px 20px 0px 40px;
+}
+
+.how-it-works_body-middle_right h3 {
+  font-weight: 700;
+  font-size: 2rem;
+}
+
+.how-it-works_body-middle_right p {
+  font-weight: 400;
+  font-size: 1.8rem;
+}
+
+/* Media View Settings */
+
+@media (min-width: 320px) and (max-width: 768px) {
```

Comment:
Your mobile screen is 320 and 480, so your media query is supposed to be 481 and 768, not 320 and 768.
Please do the necessary corrections.
I suggest you have two media queries, one for iPad(because of the images and text) and the other for laptop and larger screen sizes

Reply:
Thanks, it's the plenty deadlines that's causing limited update oh.. 

I have added extra @media settings .